3-reveal/migrations/8-IRS-Lite/1-Zambia-2020/revert/update_structure_dependencies.psql

```diff
@@ -0,0 +1,19 @@
+-- Revert reveal_irs_lite_zambia_202020:update_structure_dependencies from pg
+
+BEGIN;
+
+SET search_path TO :"schema",public;
```

Comment:
This doesn't seem to fully revert the changes introduced in this migration - at least as far as the renamed indices are concerned.  Could we please ensure that this is done?  It is important to ensure that one can apply migrations and revert them without errors.

Reply:
This is done